polaris.shopify.com/src/components/PatternPage/PatternPage.tsx

```diff
@@ -262,12 +253,6 @@ const BaseMarkdown = ({
           {children}
         </Box>
       ),
-      img: ({src, alt}) =>
-        src ? (
-          <div className={styles.ImageWrapper}>
-            <Image fill src={src} alt={alt ?? ''} />
-          </div>
-        ) : null,
```

Comment:
Not using next/image because it needs to know the width/height of an image, or an aspect ratio. We don't know those values for images specified in markdown.